refinery/core/utils.py

```diff
@@ -45,8 +49,8 @@ def update_data_set_index(data_set):
 
     logger.info('Updated data set (uuid: %s) index', data_set.uuid)
     try:
-        core.search_indexes.DataSetIndex().update_object(data_set,
-                                                         using='core')
+        core.search_indexes.DataSetIndex()\
+            .update_object(data_set, using='core')
```

Comment:
It's a bit out of scope for this pull request but I'd do something like:
```
data_set_index = core.search_indexes.DataSetIndex()
data_set_index.update_object(data_set, using='core')
```
to break up the call chain and avoid line continuation